pkg/apis/extensions/validation/worker.go

```diff
@@ -17,6 +17,7 @@ package validation
 import (
 	extensionsv1alpha1 "github.com/gardener/gardener/pkg/apis/extensions/v1alpha1"
 
+	corevalidation "github.com/gardener/gardener/pkg/apis/core/validation"
 	apiequality "k8s.io/apimachinery/pkg/api/equality"
```

Comment:
Nit: Typically, we group the imports in 3 groups: (1) core golang packages, (2) local packages, (3) external packages
```suggestion
	extensionsv1alpha1 "github.com/gardener/gardener/pkg/apis/extensions/v1alpha1"
	corevalidation "github.com/gardener/gardener/pkg/apis/core/validation"
	
	apiequality "k8s.io/apimachinery/pkg/api/equality"
```